py3status/parse_config.py

```diff
@@ -151,10 +155,17 @@ def __init__(self, config):
         self.current_token = 0
         self.line = 0
         self.line_start = 0
+        self.py3_wrapper = py3_wrapper
         self.raw = config.split('\n')
         self.container_modules = []
         self.anon_count = 0
 
+    def notify_user(self, error):
+        if self.py3_wrapper:
```

Comment:
`py3_wrapper` can be `None`? I'm confused by this wrapping of `notify_user`

Reply:
yes is you run the `python parse_config.py` good for testing etc